stl/inc/ranges

```diff
@@ -791,7 +810,8 @@ namespace ranges {
         }
 
 #if !defined(__clang__) && !defined(__EDG__) // TRANSITION, DevCom-1347136
-    private:
+    private :

```

Comment:
That one is also really bad

Reply:
It was worse without adding the following newline: :joy_cat:

```cpp
    private : template <class _Left, class _Right>
              static constexpr bool _Nothrow_plus_equal =
                  noexcept(_STD declval<_Left&>() += _STD declval<const _Right&>());
```

As I mentioned in the changelog, I thought that this single space was tolerable (especially because it's in a compiler workaround that should be removed someday). Let me know if you'd rather disable clang-format here.

Reply:
I'll go ahead and disable clang-format here in a followup PR.